### DISCUSSION TITLE
Isaac Lab v1.3.0 is out!

### DISCUSSION THREAD

**Discussion by rthaker01** [isaac-sim/IsaacLab#1454] [Announcements]
👀 Overview
This release will be a final release based on **Isaac Sim 4.2** before the transition to Isaac Sim 4.5, which will likely **contain breaking changes and no longer backwards compatible with Isaac Sim 4.2 and earlier**. In this release, we introduce many features, improvements, and bug fixes, including IMU sensors, support for various types of gymnasium spaces, manager-based perception environments, and more.

Full Changelog: https://github.com/isaac-sim/IsaacLab/compare/v1.2.0...v1.3.0

✨ New Features
Adds IMU sensor by @pascal-roth in https://github.com/isaac-sim/IsaacLab/pull/619
Add Camera Benchmark Tool and Allow Correct Unprojection of distance_to_camera depth image by @glvov-bdai in https://github.com/isaac-sim/IsaacLab/pull/976
Creates Manager Based Cartpole Vision Example Environments by @glvov-bdai in https://github.com/isaac-sim/IsaacLab/pull/995
Adds image extracted features observation term and cartpole examples for it by @glvov-bdai in https://github.com/isaac-sim/IsaacLab/pull/1191
Supports other gymnasium spaces in Direct workflow by @Toni-SM in https://github.com/isaac-sim/IsaacLab/pull/1117
Adds configuration classes for spawning different assets at prim paths by @Mayankm96 in https://github.com/isaac-sim/IsaacLab/pull/1164
Adds a rigid body collection class by @Dhoeller19 in https://github.com/isaac-sim/IsaacLab/pull/1288
Adds option to scale/translate/rotate meshes in the mesh_converter by @pascal-roth in https://github.com/isaac-sim/IsaacLab/pull/1228
Adds event term to randomize gains of explicit actuators by @MoreTore in https://github.com/isaac-sim/IsaacLab/pull/1005
Adds Isaac Lab Reference Architecture documentation by @OOmotuyi in https://github.com/isaac-sim/IsaacLab/pull/1371
🔧 Improvements
Expands functionality of FrameTransformer to allow multi-body transforms by @jsmith-bdai in https://github.com/isaac-sim/IsaacLab/pull/858
Inverts SE-2 keyboard device actions (Z, X) for yaw command by @riccardorancan in https://github.com/isaac-sim/IsaacLab/pull/1030
Disables backward pass compilation of warp kernels by @Mayankm96 in https://github.com/isaac-sim/IsaacLab/pull/1222
Replaces TensorDict with native dictionary by @Toni-SM in https://github.com/isaac-sim/IsaacLab/pull/1348
Improves omni.isaac.lab_tasks loading time by @Toni-SM in https://github.com/isaac-sim/IsaacLab/pull/1353
Caches PhysX view's joint paths when processing fixed articulation tendons by @Toni-SM in https://github.com/isaac-sim/IsaacLab/pull/1347
Replaces hardcoded module paths with __name__ dunder by @Mayankm96 in https://github.com/isaac-sim/IsaacLab/pull/1357
Expands observation term scaling to support list of floats by @pascal-roth in https://github.com/isaac-sim/IsaacLab/pull/1269
Removes extension startup messages from the Simulation App by @Mayankm96 in https://github.com/isaac-sim/IsaacLab/pull/1217
Adds a render config to the simulation and tiledCamera limitations to the docs by @kellyguo11 in https://github.com/isaac-sim/IsaacLab/pull/1246
Adds Kit command line argument support by @kellyguo11 in https://github.com/isaac-sim/IsaacLab/pull/1293
Modifies workflow scripts to generate random seed when seed=-1 by @kellyguo11 in https://github.com/isaac-sim/IsaacLab/pull/1048
Adds benchmark script to measure robot loading by @Mayankm96 in https://github.com/isaac-sim/IsaacLab/pull/1195
Switches from carb to omni.log for logging by @Mayankm96 in https://github.com/isaac-sim/IsaacLab/pull/1215
Excludes cache files from vscode explorer by @Divelix in https://github.com/isaac-sim/IsaacLab/pull/1131
Adds versioning to the docs by @sheikh-nv in https://github.com/isaac-sim/IsaacLab/pull/1247
Adds better error message for invalid actuator parameters by @lgulich in https://github.com/isaac-sim/IsaacLab/pull/1235
Updates tested docker and apptainer versions for cluster deployment by @pascal-roth in https://github.com/isaac-sim/IsaacLab/pull/1230
Removes ml_archive as a dependency of omni.isaac.lab extension by @fan-ziqi in https://github.com/isaac-sim/IsaacLab/pull/1266
Adds a validity check for configclasses by @Dhoeller19 in https://github.com/isaac-sim/IsaacLab/pull/1214
Ensures mesh name is compatible with USD convention in mesh converter by @fan-ziqi in https://github.com/isaac-sim/IsaacLab/pull/1302
Adds sanity check for the term type inside the command manager by @command-z-z in https://github.com/isaac-sim/IsaacLab/pull/1315
Allows configclass to_dict operation to handle a list of configclasses by @jtigue-bdai in https://github.com/isaac-sim/IsaacLab/pull/1227
🐛 Bug Fixes
Disables replicate physics for deformable teddy lift environment by @Mayankm96 in https://github.com/isaac-sim/IsaacLab/pull/1026
Fixes Jacobian joint indices for floating base articulations by @lorenwel in https://github.com/isaac-sim/IsaacLab/pull/1033
Fixes setting the seed from CLI for RSL-RL by @kaixi287 in https://github.com/isaac-sim/IsaacLab/pull/1084
Fixes camera MDP term name and reprojection docstrings by @Mayankm96 in https://github.com/isaac-sim/IsaacLab/pull/1130
Fixes deprecation notice for using pxr.Semantics by @Mayankm96 in https://github.com/isaac-sim/IsaacLab/pull/1129
Fixes scaling of default ground plane by @kellyguo11 in https://github.com/isaac-sim/IsaacLab/pull/1133
Fixes Isaac Sim executable on pip installation by @Toni-SM in https://github.com/isaac-sim/IsaacLab/pull/1172
Passes device from CLI args to simulation config in standalone scripts by @Mayankm96 in https://github.com/isaac-sim/IsaacLab/pull/1114
Fixes the event for randomizing rigid body material by @pascal-roth in https://github.com/isaac-sim/IsaacLab/pull/1140
Fixes the ray_caster_camera tutorial script when saving the data by @mpgussert in https://github.com/isaac-sim/IsaacLab/pull/1198
Fixes running the docker container when the DISPLAY env variable is not defined by @GiulioRomualdi in https://github.com/isaac-sim/IsaacLab/pull/1163
Fixes default joint pos when setting joint limits by @kellyguo11 in https://github.com/isaac-sim/IsaacLab/pull/1040
Fixes device propogation for noise and adds noise tests by @jtigue-bdai in https://github.com/isaac-sim/IsaacLab/pull/1175
Removes additional sbatch and fixes default profile in cluster deployment by @pascal-roth in https://github.com/isaac-sim/IsaacLab/pull/1229
Fixes the checkpoint loading error in RSL-RL training script by @bearpaw in https://github.com/isaac-sim/IsaacLab/pull/1210
Fixes pytorch broadcasting issue in EMAJointPositionToLimitsAction by @bearpaw in https://github.com/isaac-sim/IsaacLab/pull/1207
Fixes body IDs selection when computing feet_slide reward for locomotion-velocity task by @dtc103 in https://github.com/isaac-sim/IsaacLab/pull/1277
Fixes broken URLs in markdown files by @DorsaRoh in https://github.com/isaac-sim/IsaacLab/pull/1272
Fixes net_arch in sb3_ppo_cfg.yaml for Isaac-Lift-Cube-Franka-v0 task by @LinghengMeng in https://github.com/isaac-sim/IsaacLab/pull/1249
🤗 New Contributors
@riccardorancan made their first contribution in https://github.com/isaac-sim/IsaacLab/pull/1030
@glvov-bdai made their first contribution in https://github.com/isaac-sim/IsaacLab/pull/976
@kaixi287 made their first contribution in https://github.com/isaac-sim/IsaacLab/pull/1084
@lgulich made their first contribution in https://github.com/isaac-sim/IsaacLab/pull/1119
@nv-apoddubny made their first contribution in https://github.com/isaac-sim/IsaacLab/pull/1118
@GiulioRomualdi made their first contribution in https://github.com/isaac-sim/IsaacLab/pull/1163
@Divelix made their first contribution in https://github.com/isaac-sim/IsaacLab/pull/1131
@sheikh-nv made their first contribution in https://github.com/isaac-sim/IsaacLab/pull/1247
@dtc103 made their first contribution in https://github.com/isaac-sim/IsaacLab/pull/1277
@DorsaRoh made their first contribution in https://github.com/isaac-sim/IsaacLab/pull/1272
@louislelay made their first contribution in https://github.com/isaac-sim/IsaacLab/pull/1271
@LinghengMeng made their first contribution in https://github.com/isaac-sim/IsaacLab/pull/1249
@OOmotuyi made their first contribution in https://github.com/isaac-sim/IsaacLab/pull/1337
@command-z-z made their first contribution in https://github.com/isaac-sim/IsaacLab/pull/1315
@MoreTore made their first contribution in https://github.com/isaac-sim/IsaacLab/pull/1005
